plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveCompressionCodecs.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive;
+
+import io.trino.plugin.hive.metastore.StorageFormat;
+import io.trino.spi.connector.ConnectorSession;
+
+public final class HiveCompressionCodecs
+{
+    private HiveCompressionCodecs() {}
+
+    public static HiveCompressionCodec selectCompressionCodec(ConnectorSession session, StorageFormat storageFormat)
+    {
+        return HiveStorageFormat.getHiveStorageFormat(storageFormat)
+                .map(format -> selectCompressionCodec(session, format))
+                .orElse(HiveCompressionCodec.NONE);
```

Comment:
Why none for unknown format?

i think it's a behavioral change. if so, should be code-commented and highlited in the commit message

Reply:
Yeah - it is a behavioral change. I think it is fine though. This code path is used only for empty files. And compressing those does not make much sense anyway. I will add comment and change commit message.
Do you think it should make it to RN?

Reply:
I will also inline this method so it is obvious it is no used outside empty-file case

Reply:
Edit: renamed to `selectCompressionCodecForEmptyFiles` and kept as utility methods - as it is called from two source files and inline looked kinda ugly.